apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -833,6 +833,7 @@ class ScriptEditor extends React.Component {
               <input
                 value={this.state.weeklyInstructionalMinutes}
                 style={styles.input}
+                disabled={!this.state.showCalendar}
```

Comment:
So looking at your picture it looks like there is a default value that will be added to this field. That default value will still be saved even if you don't pick show calendar. It seems weird to disable it if it is filled it. I might vote for just leaving it up to the editor to be smart here and not disabling it

Reply:
hmm I asked Tess and this is what we came up with, should we start a thread on slack or something?

Reply:
Ok if this is how tess wanted it then thats fine.